pyrsh.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import paramiko
+import argparse
+import sys
+from contextlib import contextmanager
+import time
+
+def arguments():
+    parser = argparse.ArgumentParser(description='Remote command execution via ssh')
+    parser.add_argument('host', help='Hostname', type=str)
+    parser.add_argument('user', help='Your user', type=str)
```

Comment:
@GitSergeyGit Minor: can we avoid using "your" in the description? Lets provide more informative desc, like `User name to use for authentication`